andela/hc-silicon
# [Feature #157694758] Allocate Checks To Stakeholders

### What does this PR do?
Allows a team owner to allocate checks to different stakeholders.

### Description of Task to be completed?
When a team owner invites a team member, the team owner allocates the team owner checks as deemed fit by the team owner. The team member is only able to see the checks allocated by the team owner.

### How should this be manually tested?
- Start the server by running the command `./manage.py runserver` and access the URL `http://localhost:3000`. 
- To the top right, click on the email to reveal a dropdown. 
- Select 'Account Settings' then click 'Invite a Team Member' button.
- Fill in the email and department input boxes and select a check you wish to allocate a team member from the dropdown and click 'Send' button.
- An email with a login link will be sent to the team member.

### What are the relevant pivotal tracker stories?
- [Allocate checks](https://www.pivotaltracker.com/story/show/157694758)

### Screenshots
#### Inviting a team member to collaborate on check 'test3'
<img width="910" alt="screen shot 2018-06-21 at 13 16 06" src="https://user-images.githubusercontent.com/29580769/41713458-da91e1de-7555-11e8-8f65-a57b64b15a09.png">

#### Email received by team member with login link
<img width="604" alt="screen shot 2018-06-21 at 13 17 23" src="https://user-images.githubusercontent.com/29580769/41713483-f430ea72-7555-11e8-9484-33b037591fc4.png">

#### Team member sees only checks invited to ie checks 'test2 and test3'
<img width="1270" alt="screen shot 2018-06-21 at 13 19 18" src="https://user-images.githubusercontent.com/29580769/41713508-00f609f4-7556-11e8-9c89-8c3a17dfb0d0.png">

#### Checks team owner has
<img width="1255" alt="screen shot 2018-06-21 at 13 24 30" src="https://user-images.githubusercontent.com/29580769/41713650-80712bc8-7556-11e8-987f-cf2ccd62edde.png">


@Awesome94 @Felistas @tonykimathi @kzyGit @muhozi @sam-karis
